app/components/shared/inputs/TimePickerInput.vue.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Prop } from 'vue-property-decorator';
+import VueTimepicker from 'vue2-timepicker';
+import { BaseInput } from './BaseInput';
+import { ITimeMetadata } from './index';
+
+export interface ITimeInputValueMetadata {
+  HH?: string;
+  H?: string;
+  hh?: string;
+  a?: string;
+  A?: string;
+  h?: string;
+  kk?: string;
+  k?: string;
+  m?: string;
+  mm?: string;
+  s?: string;
+  ss?: string;
+}
```

Comment:
Why we need this object instead of a more traditional timestamp or Date() ?

Reply:
It looks like maybe this is the format expected by `vue2-timepicker`?  If that's the case I really think we should try to find another library soon.  😃

Reply:
We came to a decision to use milliseconds to store time and intervals. Milliseconds have no problems with serialization like a Date object and we can convert it to any format we want via moment.js